Bio/Blast/Record.py

```diff
@@ -303,11 +302,10 @@ def to_generic(self):
                 seq_parts[n] += seq
                 n += 1
 
-        generic = MultipleSeqAlignment([])
-        for (name, seq) in zip(seq_names, seq_parts):
-            generic.append(SeqRecord(Seq(seq), name))
-
-        return generic
+        records = (
+            SeqRecord(Seq(seq), name) for (name, seq) in zip(seq_names, seq_parts)
+        )
+        return MultipleSeqAlignment(records)
```

Comment:
Is it clearer to do this in one go without defining ``records`` variable?

Reply:
Then it gets too convoluted for my taste.